### DISCUSSION TITLE
Energy % for Envenom

### DISCUSSION THREAD

**Discussion by noaharogers** [Hekili/hekili#1278] [General]
As an Assassination Rogue, when I change the slider and reload the game, the recommendation for Envenom doesn't change. I could have it at 0, or 100 and envenom is always recommended as soon as its up regardless of my energy. 

Is there something I'm missing with this feature or is this how its supposed to work?

**Comment by Hekili** [comment]
It's an old feature for an older version of the Assassination priority.  Since the Assassination priority doesn't check the `envenom_pool_pct` variable any longer, it doesn't really make a difference.  I can hide it to avoid confusion.  (It really never had a meaningful impact on DPS, either.  It was just a way to decide if you wanted pooling behavior or not.)

**Comment by noaharogers** [comment]
Thanks for the reply. Good to know! I wasn't sure I was missing something.

Awesome addon, keep up the great work!
